docs/develop/localhost.md

```diff
@@ -54,7 +54,56 @@ Then, as long as the container is running, you can invoke the model as an API as
 curl -X POST http://127.0.0.1:8080/v1/models/model:predict -d '{"inputs": [[0, 0, 0, 0]]}'
 ```
 
-### On (not) using Docker
+## Setting up local dev
+
+You have three options for how to run your Truss locally:
+
+* In a Docker container
+* In a Docker container with live reload
+* Without a Docker container in the Truss directory
+
+### Running with Docker
+
+This is the standard way to run a Truss. It creates a Docker image, runs the Truss, and returns a prediction. This method closely matches production environments, but results in a slow dev loop as it rebuilds the Docker container with every change to your code.
+
+To run in Docker:
+
+```python
+tr.docker_predict({"inputs": [[0, 0, 0, 0]]})
+```
+
+### Faster dev loop with live reload
+
+You can turn on a new feature, live reload, by setting the `live_reload` property to `True` in `config.yaml`. This feature makes it faster to run a Truss in Docker in some situations.
```

Comment:
Should docs reference a feature inline as "new"? Seems like it could make it hard to track/remember to make it not new

Reply:
Removed new, though these docs will be refactored once the feature is the default behavior for docker.